index.php

```diff
@@ -225,6 +225,11 @@ function gtag(){dataLayer.push(arguments);}
 <td>Henna</td>
 <td>#2020</td>
 </tr>
+<tr>
+<td>Marissa</td>
+<td>MONDAY, FEBRUARY 10TH 2020</td>
```

Comment:
Umm, I don't think this is the proper date formatting we use

Reply:
I believe it's TWENTYMEMETEEN, year of our Lord

Reply:
y'all suck